src/Datadog.Trace/Tagging/HttpTags.cs

```diff
@@ -0,0 +1,42 @@
+using Datadog.Trace.ExtensionMethods;
+
+namespace Datadog.Trace.Tagging
+{
+    internal class HttpTags : CommonTags
+    {
+        internal static readonly IProperty<string>[] HttpTagsProperties =
+            CommonTagsProperties.Concat(
+                new Property<HttpTags, string>(Trace.Tags.Env, t => t.Environment, (t, v) => t.Environment = v),
+                new Property<HttpTags, string>(Trace.Tags.Version, t => t.Version, (t, v) => t.Version = v),
+                new Property<HttpTags, string>(Trace.Tags.HttpStatusCode, t => t.HttpStatusCode, (t, v) => t.HttpStatusCode = v),
+                new Property<HttpTags, string>(HttpClientHandlerTypeKey, t => t.HttpClientHandlerType, (t, v) => t.HttpClientHandlerType = v),
+                new Property<HttpTags, string>(Trace.Tags.SpanKind, t => t.SpanKind, (t, v) => t.SpanKind = v),
```

Comment:
I thought the `SpanKind` to be a common property, but DB spans doesn't have it... weird.

Reply:
Yeah, that could be a mistake. @lucaspimentel, thoughts?

Reply:
It looks like some language tracers set this for database queries and others (like .NET)  do not. It's not required, but it wouldn't hurt to add it and it would be more technically correct.

For context, the `span.kind` tag comes from the [OpenTracing standard](https://github.com/opentracing/specification/blob/master/semantic_conventions.md):

> Either "client" or "server" for the appropriate roles in an RPC, and "producer" or "consumer" for the appropriate roles in a messaging scenario.

Reply:
I plan to optimize the handling of SpanKind in another PR, so I'll decorate the missing spans while I'm at it

Reply:
great, so for me 👍